examples/create-react-app/yarn.lock

```diff
@@ -9728,14 +9733,22 @@ typedarray@^0.0.6:
   resolved "https://registry.yarnpkg.com/typedarray/-/typedarray-0.0.6.tgz#867ac74e3864187b1d3d47d996a78ec5c8830777"
   integrity sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=
 
-uglify-js@3.4.x, uglify-js@^3.1.4:
+uglify-js@3.4.x:
   version "3.4.9"
   resolved "https://registry.yarnpkg.com/uglify-js/-/uglify-js-3.4.9.tgz#af02f180c1207d76432e473ed24a28f4a782bae3"
   integrity sha512-8CJsbKOtEbnJsTyv6LE6m6ZKniqMiFWmm9sRbopbkGs3gMPPfd3Fh8iIA4Ykv5MgaTbqHr4BaoGLJLZNhsrW1Q==
   dependencies:
     commander "~2.17.1"
     source-map "~0.6.1"
 
```

Comment:
Do we know why this time uglify produced 2 entries?

Reply:
It's removing an entry, looks it like was added [here](https://github.com/Shopify/polaris-react/commit/8f7c64bde4f501de9b2c20f82b4182c35172659a#diff-8ee2343978836a779dc9f8d6b794c3b2R17598). To quiet `audits`, but a version was probably bumped to be in semver range of `uglify-js@3.4.x:`